07-react-1/classworks/Emelyanov_Egor/src/pages/home/getSubredditInfo.js

```diff
@@ -0,0 +1,12 @@
+import { setSubredditData } from '../../store/actions/subredditData';
```

Comment:
`getSubredditInfo` should be in `entity.actions.js` file. the structure should be:
![image](https://user-images.githubusercontent.com/23057064/89271622-a35ba080-d645-11ea-9965-6780ee1cc43c.png)